tests/regression/00-sanity/34-hoare-over-addrs.c

```diff
@@ -0,0 +1,20 @@
+#include <assert.h>
+
+int main() {
+  int arr[10];
+  int r; // rand
+  int *p;
+  if (r) {
+    p = &arr[1];
+    assert(1); // extra node in cfg
+  }
+  else {
+    p = &arr[2];
+    assert(1); // extra node in cfg
+  }
+  // TODO: even though we use Hoare for addresses per-varinfo
```

Comment:
Right now it's also manual, but I think I can make this one automatic using an assertion on possible points to.

Reply:
I actually had to disable this new more precise behavior with Hoare sets in each bucket to everything force-joined in each bucket to stay compatible with #808. Since strings are not `leq` of each other, a Hoare set would keep them all, even with limiting enabled.